k8s/mariadb/deployer/common.sh

```diff
@@ -0,0 +1,87 @@
+#!/bin/bash
```

Comment:
JFYI: I'm working to add a new `x-google-marketplace` property, so we will be able to drop this code.

https://github.com/GoogleCloudPlatform/marketplace-k8s-app-tools/pull/337

Reply:
Ok, thank you